tests/test_pytypes.cpp

```diff
@@ -11,7 +11,7 @@
 
 #include <utility>
 
-namespace external {
+PYBIND11_NAMESPACE_BEGIN(PYBIND11_NS_VISIBILITY(external))
```

Comment:
nit Er, hiding a namespace `external` smells odd.

Can we use a better name here? e.g. `feaux_external`, with a comment of what is going on?